manifests/s/SteelSeries/GG/4.0.0/SteelSeries.GG.locale.en-US.yaml

```diff
@@ -0,0 +1,34 @@
+# Created using YamlCreate.ps1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+
+PackageIdentifier: SteelSeries.GG
+PackageVersion: 4.0.0
+PackageLocale: en-US
+Publisher: SteelSeries
```

Comment:
```suggestion
Publisher: SteelSeries ApS
```

Reply:
My bad, it was like that in the 3.1.0 manifest as well as for Engine, so I didn't look further.